angular-client/src/api/data.api.ts

```diff
@@ -0,0 +1,5 @@
+import { urls } from './urls';
+
+export const getDataByDataTypeName = async (dataTypeName: string) => {
```

Comment:
why is this one async and awaits the fetch call, but the rest of the apis just return promises

Reply:
thats a good question

Reply:
oh right i was trying to fuck around with getting error messages to come up and this was me trying to get it to work unsucessfully